profile_inference.py

```
@@ -695,6 +713,605 @@ def run_benchmark_mode(dit_handler, llm_handler, args, timer: PreciseTimer):
+        print(f"  recommended_backend: {gpu_config.recommended_backend}")
+        print(f"  lm_backend_restriction: {gpu_config.lm_backend_restriction}")
+        print(f"  offload_to_cpu: {gpu_config.offload_to_cpu_default}")
+        print(f"  offload_dit_to_cpu: {gpu_config.offload_dit_to_cpu_default}")
+        print(f"  quantization: {gpu_config.quantization_default}")
+        print(f"  max_duration_with_lm: {gpu_config.max_duration_with_lm}s")
+        print(f"  max_duration_without_lm: {gpu_config.max_duration_without_lm}s")
+        print(f"  max_batch_with_lm: {gpu_config.max_batch_size_with_lm}")
+        print(f"  max_batch_without_lm: {gpu_config.max_batch_size_without_lm}")
+
+        # ---- Test 1: Default configuration ----
+        print(f"\n  --- Variant: default ---")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)